### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by mribeirodantas**
- Fix minor typo and trailing white spaces in README
- Fix minor typos in the training material
- Replace Ciao by Olá in example with params

### CODE REVIEW

File: asciidocs/modules.adoc [hunk 1]
```diff
@@ -199,7 +199,7 @@ workflow  {
 }
 ----
 
-If a process defines two or more output channels, each channel can be accessed by indexing the `.out` attribute, e.g., `.out[0]`, `.out[1]`, etc. In our example we only have the `[0]`'th output:
+If a process defines two or more output channels, each channel can be accessed by indexing the `.out` attribute, e.g., `.out[0]`, `.out[1]`, etc. In our example we only have the `[0]'th` output:
```
Comment (ewels) [inline asciidocs/modules.adoc:202]: Slightly controversial, but I'm happy if there's a reason (eg. it's easier to read this way)...

**Review by ewels — APPROVED** [review]
LGTM!

File: asciidocs/modules.adoc (hunk 1 above)
Comment (mribeirodantas) [reply asciidocs/modules.adoc:202]: Before, it rendered like this 😅 
Screen Shot 2022-09-19 at 02 00 20
Now:
Screen Shot 2022-09-19 at 02 01 06

I'm fine if you want to suggest some other way. I just found a similar way that rendered correctly 😄

Comment (ewels) [reply asciidocs/modules.adoc:202]: Then yeah, that's a good reason :)

**Pull request merged**